src/buggy_code.py

```
@@ -0,0 +1,56 @@
+        else:
+            return False
+    
+    def calculate_age(self, birth_date):
+        try:
+            age = datetime.now().year - birth_date.year
+            return age
+        except:
+            pass
+    
+    def process_data(self, data):
+        l = []
```

Ruff

Ambiguous variable name: `l`

(E741)